db/Models/news.models.js

```diff
@@ -40,9 +40,39 @@ const selectArticle = (article_id) => {
   });
 };
 const selectArticles = () => {
-  let queryString = `SELECT * FROM articles`;
+  let queryString = ` SELECT 
+        articles.article_id,
+        articles.author,
+        articles.title,
+        articles.topic,
+        articles.created_at,
+        articles.votes,
+        articles.article_img_url,
+        COUNT(comments.comment_id)::INTEGER AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC`;
 
   return db.query(queryString).then((data) => {
+    // console.log(data);
+
+    if (data.rows.length === 0) {
+      return Promise.reject({ status: 400, msg: "Bad request" });
+    } else {
+      return data.rows;
+    }
+  });
+};
+const selectCommentByArticle = (article_id) => {
+  let queryString = `SELECT * FROM comments`;
+  let queryValues = [];
+
+  if (Number(article_id)) {
```

Comment:
This is fine, but also would PSQL throw us an error if we tried to put anything other than a number in for an article_id?